### PR TITLE
New Locale and InputValue contexts, IdentityContext tweaks

### PR DESCRIPTION
**Pull request by sdirosa**
We added two new GlobalContexts to the base taxonomy schema:

- `LocaleContext`, used to keep track of the current language for multi-language applications
- `InputValueContext`, pushed along InputChangeEvents to send the value of a specific input element

Other changes:  
- Added a `trackValue` option to `TrackedInputContext` and `TrackedInput` for React SDK. Default to `false`.
- Added a `trackValue` option to `TrackedTextInput` for React Native SDK. Default to `false`.
- Added a `trackValue` option to `TrackedSwitch` for React Native SDK. Default to `false`.
- Added a `trackValue` attribute to `tagInput`'s `trackBlurs` options for Browser SDK. Default to `false`.
- Created a new Plugin to factor `LocaleContext` via a developer specified function.
- Standardized all schema descriptions to be consistent.
- https://github.com/objectiv/objectiv-analytics/pull/993

### CODE REVIEW

File: schema/base_schema.json5 [hunk 1]
```diff
@@ -184,6 +184,20 @@
         }
       }
     },
+    "InputValueContext": {
+      "description": "A GlobalContext containing the value of a single input element. Multiple InputValueContexts may be present in Global Contexts at the same time.",
+      "parents": ["AbstractGlobalContext"],
+      "properties": {
+        "value": {
+          "description": "The value of the input element",
```
Comment (vincenthoogsteder) [inline schema/base_schema.json5:192]: @sdirosa super minor but add "." at the end behind "element" so it's consistent with the rest?

File: schema/base_schema.json5 [hunk 2]
```diff
@@ -184,6 +184,20 @@
         }
       }
     },
+    "InputValueContext": {
+      "description": "A GlobalContext containing the value of a single input element. Multiple InputValueContexts may be present in Global Contexts at the same time.",
```
Comment (vincenthoogsteder) [inline schema/base_schema.json5:188]: @sdirosa would it make more sense to call this value maybe?

Comment (sdirosa) [reply schema/base_schema.json5:188]: It's called `value`. This is the description of the Context itself, it's used to generate documentation blocks.

Comment (vincenthoogsteder) [reply schema/base_schema.json5:188]: my bad! aligning in the docs

Comment (sdirosa) [reply schema/base_schema.json5:188]: Cool, to be clear: this context has `id` and `value` properties. As a convention I'm setting the `id` fields to the same value of the corresponding Input Location Context. The `value` is just the element's value.

File: schema/base_schema.json5 (hunk 1 above)
Comment (sdirosa) [reply schema/base_schema.json5]: Done

File: schema/base_schema.json5 [hunk 3]
```diff
@@ -245,17 +258,20 @@
       }
     },
     "IdentityContext": {
-      "description": "A Global Context to track the identity of users across sessions, platforms, devices. Multiple can be present.",
+      "description": "A Global Context to track the identity of users across sessions, platforms, devices. Multiple can be present.\n\
+        The `id` field is used to specify the type of identification e.g. backend, md5(email), supplier_cookie.\n\
```
Comment (jansentom) [inline schema/base_schema.json5:262]:
```suggestion
        The `id` field is used to specify the scope of identification e.g. backend, md5(email), supplier_cookie, etc.\n\
```

Comment (sdirosa) [reply schema/base_schema.json5:262]: Will regenerate again in a moment 👌🏼

**Review by borft — APPROVED** [review]

**Pull request merged**